dts/bindings/clock/clock.yaml

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2019 Derek Hageman <hageman@inthat.cloud>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+title: Base clock provider binding
+version: 0.1
+
+description: >
+    This is the base binding for clock providers.
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      generation: define
+
+    clock-frequency:
```

Comment:
`clock-frequency` should stay in fixed-clock.yaml.

Reply:
Ok.